frontend/src/router.js

```diff
@@ -0,0 +1,25 @@
+import Vue from "vue";
+import Router from "vue-router";
+import Home from "./views/Home.vue";
+import Lobby from "./views/Lobby.vue";
+
+Vue.use(Router);
+
+export default new Router({
+    routes: [
+        {
+            path: "/",
+            name: "home",
+            component: Home
+        },
+        {
+            path: "/lobby",
+            redirect: "/"
```

Comment:
does this mean route back to /, meaning route to "home"?

Reply:
Yes